README.md

```diff
@@ -34,6 +34,8 @@ foo _ = --
 
 Usages of `# Type` should be replaced with `Row Type`.
 
+<!-- Before and after examples for all of these changes would be nice, even if it's just a simple substitution. -->
+
```

Comment:
I can't think of an example. And in the ones that I did for `# Type` and `RowList`, I just added a kind signature to it anyways, which is already covered below.